DataDog/dd-agent
# [trace-agent] adding output of 'trace-agent -info'

### What does this PR do?

When calling `datadog-agent info`, this patch makes a subcall to `trace-agent -info` to show trace agent (APM) status.

### Motivation

It's easier to monitor the agent status this way, provides a quick way to know what's happening.

### Testing Guidelines

- Check that  `datadog-agent status` display relevant info.
- Check the return code of that command (should be 0 on  success, obviously)

### Additional Notes

- Wondering wether Debian support is enough or if all platforms should be done...
- What happens when the agent is not running, should we report an error (return code 1) or return 0 as APM is not part of the base, core install

*IMPORTANT* -> must **NOT** ship before: https://github.com/DataDog/datadog-trace-agent/pull/214
